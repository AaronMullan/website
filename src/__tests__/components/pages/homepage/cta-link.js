import React from 'react'
import renderer from 'react-test-renderer'
import { useStaticQuery } from 'gatsby'
import CtaLink from '~components/pages/homepage/cta-link'

describe('Components : Pages : Homepage : Press logos', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <CtaLink to={'/data'}>
        This is a test CtaLink
      </CtaLink>
    ).toJSON()
    expect(tree).toMatchSnapshot()

    const treeCentered = renderer.create(
      <CtaLink to={'/data'} centered>
        This is a centered CtaLink
      </CtaLink>
    ).toJSON()
    expect(treeCentered).toMatchSnapshot()
  })
})
