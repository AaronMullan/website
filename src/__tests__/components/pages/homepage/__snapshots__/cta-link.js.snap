// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Homepage : Press logos renders correctly 1`] = `
<a
  className="cta "
  href="/data"
>
  This is a test CtaLink
  <span
    aria-hidden={true}
    className="arrow"
  >
     
    →
  </span>
</a>
`;

exports[`Components : Pages : Homepage : Press logos renders correctly 2`] = `
<a
  className="cta centered"
  href="/data"
>
  This is a centered CtaLink
  <span
    aria-hidden={true}
    className="arrow"
  >
     
    →
  </span>
</a>
`;
